app/views/dashboard/_comment_moderate.html.erb

```diff
@@ -0,0 +1,11 @@
+<% if comment.status == 4 %>
```

Comment:
These are changed from any other pr.
Can you please remove these changes if they are irrelevant?

Reply:
Yep, will do